pkg/controllers/localbuild/argo.go

```diff
@@ -15,6 +22,14 @@ import (
 //go:embed resources/argo/*
 var installArgoFS embed.FS
 
+const (
+	argocdDevModePassword         = "developer"
+	argocdAdminSecretName         = "argocd-secret"
+	argocdInitialAdminSecretName  = "argocd-initial-admin-secret"
+	argocdInitialAdminPasswordKey = "argocd-initial-admin-secret"
+	argocdNamespace               = "argocd"
```

Comment:
These are already defined in other packages I think. Either export them or move them to globals or APIs

Reply:
I removed 2 non used constants and others are non declared in another go file or package. See: 78e1e40